decision-policy.php

```diff
@@ -14,27 +14,37 @@
-		<p>During discussion on a topic, participants are welcome to raise objections freely to help ensure that all available information can be considered and contribute to the best possible decision. However, when the chairs issue a Call for Consensus, objections should not be raised unless the individual strongly believes the decision is the wrong one in spite of discussion, and the individual cannot "live with" the decision. Compromise on points that the individual considers suboptimal but can "live with" is an essential part of group decisions that must meet various requirements. </p>
+		<p>During discussion on a topic, participants are welcome to raise objections freely to help ensure that all available information can be considered and contribute to the best possible decision. However, when the chairs issue a Call for Consensus (CfC), objections should not be raised unless the individual strongly believes the decision is the wrong one in spite of discussion, and the individual cannot "live with" the decision. Compromise on points that the individual considers suboptimal but can "live with" is an essential part of group decisions that must meet various requirements.  Objections from people who raised their concerns to the group during discussions may be given more consideration than objections from people who had opportunity and chose to not participate in those discussions.</p>
 		<ol>
 			<li> Discussion on a topic proceeds until the chairs believe that all points of view have been expressed and the group has considered the variety of information presented. Depending on the topic, this discussion may take a couple of days or a couple of weeks, or more. 
 				<ol>
-					<li> Discussion can take place in any <a href="./communication">recognized channel of the Working Group</a> including email on the AG mailing list, comment threads for GitHub issues or pull requests, or on Working Group calls. </li>
+					<li> Discussion can take place in any <a href="./communication">recognized channel of the Working Group</a> including email on the public AG mailing list, AG surveys, comment threads for GitHub issues or pull requests, or on Working Group calls. </li>
 				</ol>
 			</li>
 			<li> When the chairs believe that the group is ready to come to a decision they announce a Call for Consensus by email to the Working Group's mailing list. The Call must remain open for a minimum of two working days. 
 				<ol>
-					<li> The Call is open to responses from all group members. </li>
+					<li> The Call is open to responses from all <a href="https://www.w3.org/groups/wg/ag/participants">group participants</a>. </li>
```

Comment:
Is there a difference between member and participant? Are you going back to "in good standing"?